Flutter/pdf-viewer/text-selection.md

```diff
@@ -74,6 +74,28 @@ class _HomePageState extends State<HomePage> {
 {% endhighlight %}
 {% endtabs %}
 
+## Customize the visibility of the text selection context menu
+
+The `canShowTextSelectionMenu` property allows the user to customize the visibility of the built-in text selection context menu. The following code example explains this.
```

Comment:
Please mention what we are doing in this example in words also.

<img width="367" alt="image" src="https://github.com/syncfusion-content/flutter-docs/assets/65169077/f891eba6-0beb-419d-8feb-f662a4c38038">

Reply:
Added necessary explanation.